Firestore/core/src/firebase/firestore/remote/watch_stream.mm

```diff
@@ -73,7 +73,7 @@
 }
 
 void WatchStream::NotifyStreamOpen() {
-  delegate_bridge_.NotifyDelegateOnOpen();
+  callback_->OnWatchStreamOpen();
```

Comment:
Judging by the implementation, callback_ needs to be notnull. Consider asserting that in the ctor.